src/applications/representative-appoint/containers/NextStepsPage.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import { getFormSubtitle } from '../utilities/helpers';
+import AddressBlock from '../components/AddressBlock';
+import ContactCard from '../components/ContactCard';
+import NeedHelp from '../components/NeedHelp';
+
+export default function NextStepsPage() {
+  const { data: formData } = useSelector(state => state.form);
+  const address = {
+    address1: (
+      formData['view:selectedRepresentative']?.addressLine1 || ''
+    ).trim(),
+    address2: (
+      formData['view:selectedRepresentative']?.addressLine2 || ''
+    ).trim(),
+    address3: (
+      formData['view:selectedRepresentative']?.addressLine3 || ''
+    ).trim(),
+    city: (formData['view:selectedRepresentative']?.city || '').trim(),
+    state: (formData['view:selectedRepresentative']?.stateCode || '').trim(),
+    zip: (formData['view:selectedRepresentative']?.zipCode || '').trim(),
+  };
+  const isOrg =
+    formData['view:selectedRepresentative']?.type === 'organization';
+  const isAttorneyOrClaimsAgent =
+    formData.repTypeRadio === 'Attorney' ||
+    formData.repTypeRadio === 'Claims Agent';
+
+  const getRepType = () => {
+    if (isAttorneyOrClaimsAgent) {
+      return formData.repTypeRadio.toLowerCase();
+    }
+
+    return 'VSO representative';
+  };
+
+  const getOrgName = () => {
+    if (isOrg) {
+      return formData['view:selectedRepresentative'].name;
+    }
+
+    if (isAttorneyOrClaimsAgent) {
+      return null;
+    }
+
+    const id = formData?.selectedAccreditedOrganizationId;
+    const orgs =
+      formData['view:selectedRepresentative']?.attributes
+        .accreditedOrganizations.data;
+    let orgName = null;
+
+    if (id && orgs) {
```

Comment:
You should be able to use the `.find` array method

Reply:
Nice. Updated.

Reply:
Thanks, I think I prefer doing it this way.